SymSettings.cfg

```diff
@@ -0,0 +1,75 @@
+### MAIN ###
```

Comment:
You should remove this file, the goal is that all the settings files are in the folder now instead